README.md

```diff
@@ -333,11 +333,16 @@ watermark: {
     - `markers.attrY:` _(string, default: 'cy')_ Attribute to position the marker on the Y-Axis
     - `markers.attrXDelta:` _(integer, default: 0)_ Left relative margin of the marker
     - `markers.attrYDelta:` _(integer, default: 0)_ Top relative margin of the marker
+    - `markers.groupby:` _(function(groupedData))_ Given groupedData (the current row), function that returns a value that we group markers by.
+    - `markers.rollup:` _(function(groupedData))_ Given groupedData (the current row), function that returns a value that we would use for every marker.
     - `markers.tooltipClassName:` _(string, default: 'mt-map-tooltip popover bottom')_ Class name of the tooltip used for markers (we're using bootstrap).
     - `markers.tooltip:` _(function(groupedData))_ Function that returns html that we would use as content for the tooltip. We recommend you to use the bootstrap popover..
     - `markers.attr:` _(object)_ Markers attributes (same naming as SVG attributes).
         - `markers.attr.fill:` _(ScaledValue)_ Marker background color.
-        - `markers.attr.r:` _(ScaledValue)_ Marker radius.
+        - `markers.attr.r.max:` _(ScaledValue)_ Maximum radius.
+        - `markers.attr.r.min:` _(ScaledValue)_ Minimum radius
+        - `markers.attr.r.rollup:` _(function(groupedData), default: values.length)_ Function for the values we're attaching to the radius. Defaults to ``values.length``
+        - `markers.attr.r.transform:` _(function(value), default: value)_ Function for changing the value for radius of the marker (eg ``Math.log(value)``)
```

Comment:
Sorry, I'm still confused :(

I know ``markers.attr.r`` has ``markers.attr.r.transform`` property as it was missing and I added it ;)  However, it can not take the same signature as ``countries.attr.transform`` because that accepts the value *and* rawDataset.  The signature for ``markers.attr.r.transform`` is still just the ``value``.   Maybe we need to chat real time to see what you're getting at? I wanna fix it!!